proj/hps_accel/Makefile

```diff
@@ -24,7 +24,7 @@ DEFINES += ACCEL_CONV
 #DEFINES += DUMP_LAYER=20
 
 # Uncomment this line to use software defined CFU functions in software_cfu.cc
-DEFINES += CFU_SOFTWARE_DEFINED
+#DEFINES += CFU_SOFTWARE_DEFINED
```

Comment:
This commit's contents don't seem to match its message, I guess a minor rebasing mistake?

Reply:
I'll reword the commit message